xbmc/addons/AddonManager.cpp

```diff
@@ -607,6 +607,8 @@ bool CAddonMgr::FindAddon(const std::string& addonId,
   FindAddons(installedAddons, "special://xbmcbin/addons");
```

Comment:
from my kodi.log: `special://xbmcbin/` is mapped to: `/opt/matrix/lib/kodi`
that means: `special://xbmcbin/addons` is mapped to `/opt/matrix/lib/kodi/addons`